ros2_serial_example/config/launch_px4_serial_to_ros2_bridge_params.yaml

```diff
@@ -0,0 +1,153 @@
+backend_comms: udp             # Which backend to use for communications;
```

Comment:
For what it is worth, I've never had to do this before.  I really think the `default_ros2_to_serial_bridge_params.yaml` should just work as-is.  At the very least, we should understand why it does not.

Reply:
It's something under review https://github.com/orgs/ros2/projects/13#card-27860106

Reply:
Ah right, good call.  Now I recall what I've done elsewhere, which is to use the `yaml` library in launch files to read this in and re-root it.  An example of what I'm talking about is here: https://github.com/ros-drivers/velodyne/blob/8091dda8fe6fb1869963187ce593d11e5f80062d/velodyne_driver/launch/velodyne_driver_node-VLP16-composed-launch.py#L51

I'm somewhat inclined to replicate that, though that would necessitate adding a launch file to this project with this PR.  What do you think about that?

Reply:
Thanks for the pointer @clalancette. Resolved ;)